release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2630,6 +2630,7 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
 
 * Due to the inclusion of old images in some image indexes, running `oc adm catalog mirror` and `oc image mirror` might result in the following error: `error: unable to retrieve source image`. As a temporary workaround, you can use the `--skip-missing` option to bypass the error and continue downloading the image index. For more information, see link:https://access.redhat.com/solutions/6975305[Service Mesh Operator mirroring failed].
 
+* It is not possible to create a macvlan on the physical function (PF) when a virtual function (VF) already exists. This issue affects the Intel E810 NIC. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2120585[BZ#2120585])
```

Comment:
Not sure if it is MacVLan or macvlan?

Reply:
@jeana-redhat I did a search on docs before adding to see what it is, and it seems to be macvlan across the docs.  I think we are OK with macvlan as I think it well understood this is the CNI plugin original content already merged to 4.12 RN so wold like to keep this the same.